### DISCUSSION TITLE
Can we run the code in interactive window?

### DISCUSSION THREAD

**Discussion by yangsun-h20** [coder/code-server#4966] [Q&A]
Hi I installed code-server but in the web VSCode version, I tried to run the code in interactive window but got error message with "Command 'Jupyter: Create Interactive Window' resulted in an error (command 'jupyter.createnewinteractive' not found)"

**Comment by jsjoeio** [comment]
Is this the same issue you're facing? https://github.com/coder/code-server/issues/4964
